WallPaperDev/WallPaperDev/Views/Controllers/EditTextLabelViewController.swift

```diff
@@ -179,6 +241,44 @@ extension EditTextLabelViewController {
 
     toolBar.setItems(items, animated: true)
   }
+
+  private func setupTimeDateLayers() {
+    let timeFormatter = DateFormatter()
+
+    let localSetting = DateFormatter.dateFormat(fromTemplate: "j", options: 0, locale: Locale.current)
```

Comment:
what exactly is the template for "j"?

Reply:
It's for the hours of the day. Since I wanted to include whether or not the user chose either 12 or 24 hours format. 

In `DateFormatter` "a" represent the AM or PM. So if the hours format includes an "a" character, it means that the user has 12 hours time format chosen.

Reply:
Ah! Nice